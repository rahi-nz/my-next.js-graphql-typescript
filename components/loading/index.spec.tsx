import React from "react";
import renderer from "react-test-renderer";
import Loading from "./index";

describe("<Loading/>", () => {
  let rootElement;
  let testRenderer;

  // eslint-disable-next-line prefer-const,react/display-name
  rootElement = () => <Loading />;
  // eslint-disable-next-line prefer-const
  testRenderer = renderer.create(rootElement());

  it("Should render correctly", () => {
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
