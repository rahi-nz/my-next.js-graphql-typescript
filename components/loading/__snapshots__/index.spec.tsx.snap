// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> Should render correctly 1`] = `
<div
  className="style__Container-sc-3o03qe-0 eOGrKg"
>
  <img
    alt="myAppLoading"
    className="style__LoadingImg-sc-3o03qe-1 hGRdTz"
    src="/loading.gif"
  />
</div>
`;
